main.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Aug 15 14:17:16 2019
+
+@author: j-bd
+
+To launch the training process:
+python main.py -c train -of "/path/to/kaggle/data/folder/"
+-pf "/path/to/your/project/folder/" -w "/path/to/your/file.h5"
+All these arguments are mandatory. Regarding the weights choice "-w". You either
+can provide:
+    the path to a file "wights.h5"
+    or enter "coco" or "imagenet" to make automaticaly downloading
+
+To lauch detection:
+python main.py -c detection -of "/path/to/kaggle/data/folder/"
+ -pf "/path/to/your/project/folder/" -w "/path/to/your/file.h5"
+All these arguments are mandatory.
+"""
+
+import ship_functions
+
+
+def detection(args):
+    '''Launch the detection for selected images'''
+    ORIGIN_DIR = args.origin_folder
+    ORIGIN_TEST_DIR = ORIGIN_DIR + "test_v2/"
```

Comment:
1. Use `os.path` or `pathlib`.
2. Most of the time avoid writing the trailing `/` in your paths, it does not break anything but it's the most common usage. (Although it makes a difference in certain situations like when using symbolic links or rsync)

Reply:
Same comment for several other places in your code